tests/app/dao/test_organisation_dao.py

```diff
@@ -119,17 +119,57 @@ def test_update_organisation_domains_lowercases(
     assert {domain.domain for domain in organisation.domains} == expected_domains
 
 
-def test_add_service_to_organisation(notify_db, notify_db_session, sample_service, sample_organisation):
+def test_update_organisation_does_not_update_the_service_org_type_if_org_type_is_not_provided(
+    sample_service,
+    sample_organisation,
+):
+    sample_service.organisation_type = 'local'
+    sample_organisation.organisation_type = 'central'
+
+    sample_organisation.services.append(sample_service)
+    db.session.commit()
+
+    assert sample_organisation.name == 'sample organisation'
+
+    dao_update_organisation(sample_organisation.id, name='updated org name')
+
+    assert sample_organisation.name == 'updated org name'
+    assert sample_service.organisation_type == 'local'
```

Comment:
It's probably a good idea to add a check that `ServiceHistory` is updated as well.

Reply:
The service doesn't get updated in this test, but I've added versioning to the `dao_add_service_to_organisation` function and a test for `ServiceHistory` in `test_add_service_to_organisation`